src/DynamoCoreWpf/Interfaces/IWorkspaceViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Dynamo.Models;
+using System;
+
+namespace Dynamo.Wpf.Interfaces
+{
+    public interface IWorkspaceViewModel
+    {
+        WorkspaceModel Model { get; }
```

Comment:
I wanted to use `IWorkspaceModel` here, but there are a lot of places, where it's needed to use `WorkspaceModel` with its' properties and events.